hazelcast/src/main/java/com/hazelcast/map/impl/MapDataSerializerHook.java

```diff
@@ -52,8 +53,9 @@
     public static final int QUERY_RESULT = 10;
     public static final int EVICT_BACKUP = 11;
     public static final int CONTAINS_KEY = 12;
+    public static final int KEY_ITERATOR = 13;
 
-    private static final int LEN = CONTAINS_KEY + 1;
+    private static final int LEN = KEY_ITERATOR + 1;
 
```

Comment:
I'm missing some javadocs on the key classes and their key methods (but I guess it's still work in progress):
- AbstractMapPartitionIterator
- RecordStore
- CachePartitionIterator
- etc.